modules/core/src/main/scala/lucuma/odb/api/schema/TargetMutation.scala

```diff
@@ -225,8 +236,8 @@ trait TargetMutation extends TargetScalars {
       name        = "createTarget",
       fieldType   = TargetType[F],
       description = "Creates a new target according to the provided parameters.  Only one of sidereal or nonsidereal may be specified.".some,
-      arguments   = List(ArgumentTargetCreate),
-      resolve     = c => c.target(_.insert(c.arg(ArgumentTargetCreate)))
+      arguments   = List(ProgramIdArgument, ArgumentTargetCreate),
```

Comment:
Took the `programId` out of the target creation arguments.  You should be able to use the same parameters to create targets in multiple programs, I thought.